docs/core/docker/build-container.md

```diff
@@ -122,205 +135,194 @@ Counter: 4
 
-> [!NOTE]
+> [!IMPORTANT]
 > Any parameters after `--` are not passed to the `dotnet run` command and instead are passed to your application.
 
 ## Publish .NET Core app
 
-Before you add your .NET Core app to the Docker image, publish it. You want to make sure that the container runs the published version of the app when it's started.
-
-From the working folder, enter the *app* folder with the example source code and run the following command:
+Before adding the .NET Core app to the Docker image, first it must be published. It is best to have the container run the published version of the app. To publish the app, execute the following command:
 
 ```dotnetcli
 dotnet publish -c Release
 ```
 
-This command compiles your app to the *publish* folder. The path to the *publish* folder from the working folder should be `.\app\bin\Release\netcoreapp3.1\publish\`
+This command compiles your app to the *publish* folder. The path to the *publish* folder from the working folder should be `.\App\bin\Release\netcoreapp3.1\publish\`
+
+#### [Windows](#tab/windows)
+
+From the *App* folder, get a directory listing of the publish folder to verify that the *NetCore.Docker.dll* file was created.
 
-From the *app* folder, get a directory listing of the publish folder to verify that the *myapp.dll* file was created.
+```powershell
+dir .\bin\Release\netcoreapp3.1\publish\
 
-```console
-> dir bin\Release\netcoreapp3.1\publish
+    Directory: C:\Users\dapine\App\bin\Release\netcoreapp3.1\publish
 
-    Directory:  C:\docker-working\app\bin\Release\netcoreapp3.1\publish
 
-01/09/2020  11:41 AM    <DIR>          .
-01/09/2020  11:41 AM    <DIR>          ..
-01/09/2020  11:41 AM               407 myapp.deps.json
-01/09/2020  12:15 PM             4,608 myapp.dll
-01/09/2020  12:15 PM           169,984 myapp.exe
-01/09/2020  12:15 PM               736 myapp.pdb
-01/09/2020  11:41 AM               154 myapp.runtimeconfig.json
+Mode                LastWriteTime         Length Name
+----                -------------         ------ ----
+-a----        4/27/2020   8:27 AM            434 NetCore.Docker.deps.json
+-a----        4/27/2020   8:27 AM           6144 NetCore.Docker.dll
+-a----        4/27/2020   8:27 AM         171520 NetCore.Docker.exe
+-a----        4/27/2020   8:27 AM            860 NetCore.Docker.pdb
+-a----        4/27/2020   8:27 AM            154 NetCore.Docker.runtimeconfig.json
 ```
 
-If you're using Linux or macOS, use the `ls` command to get a directory listing and verify that the *myapp.dll* file was created.
+#### [Linux](#tab/linux)
+
+Use the `ls` command to get a directory listing and verify that the *NetCore.Docker.dll* file was created.
 
 ```bash
 me@DESKTOP:/docker-working/app$ ls bin/Release/netcoreapp3.1/publish
-myapp.deps.json  myapp.dll  myapp.pdb  myapp.runtimeconfig.json
+NetCore.Docker.deps.json  NetCore.Docker.dll  NetCore.Docker.pdb  NetCore.Docker.runtimeconfig.json
 ```
 
```

Comment:
You're missing the `---` end tabs region syntax. Also, the guide says that the tabs should be single `#` headers. https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/tabbed-conceptual?branch=master

I don't know if the `#` header makes a difference, but in general we do that. If we use some automation to change them in the future, it would be good to have them consistent. 😄

Reply:
Great catch, I wish that this feature was automated and validated... Also, the `#` was giving me warnings in the markdown lint (as it treated those as H1's), I know that they work regardless of H1-6, as I used them in the Azure docs a lot.

Reply:
OK keep it then :D I have disabled the warning in the linter then reenabled it later in the article, but that is more work than necessary. I like your approach better 😄